syntax  = "proto3";

option go_package = "helloworld/hello";

service HelloService {
    rpc Hello(HelloRequest) returns (HelloResponse) {}
}

message HelloRequest {
    string name = 1;
}

message HelloResponse {
    string response = 1;
}